src/entry/Entry.tsx

```diff
@@ -36,15 +37,32 @@ const Entry: React.FC<EntryProps> = ({ match }) => {
           type={FreeTextType.PATHWAY}
           includeTitle={true}
         />
+        <FeaturesView
+          data={entryData}
+          types={[
```

Comment:
Can we convert `types` to an enum

Reply:
Fair point! I think we need to find a consistent way to name files. Most under /types start with  lower case and have the `types` suffix so I adopted that, but `EntrySections.ts` doesn't follow that pattern